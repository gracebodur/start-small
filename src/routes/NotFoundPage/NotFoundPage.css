.Not-found {
    margin-top: 30px;
}