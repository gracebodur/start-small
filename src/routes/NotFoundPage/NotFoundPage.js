import React from 'react'
import './NotFoundPage.css'

function NotFoundPage() {
    return (
        <section className='Not-found'>
            <h2>404-Page not found</h2>
            <p>You may want to start over</p>
        </section>
    )
}

export default NotFoundPage