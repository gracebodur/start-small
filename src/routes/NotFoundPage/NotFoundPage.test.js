import React from "react";
import toJson from "enzyme-to-json";
import NotFoundPage from "./NotFoundPage";

describe(`NotFoundPage component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a NotFoundPage by default", () => {
    const wrapper = <NotFoundPage />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the NotFoundPage given props", () => {
    const wrapper = <NotFoundPage {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
