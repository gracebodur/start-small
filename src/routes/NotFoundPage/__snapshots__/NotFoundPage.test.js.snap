// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component renders a NotFoundPage by default 1`] = `undefined`;

exports[`NotFoundPage component renders the NotFoundPage given props 1`] = `undefined`;
